Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:16.04
+
+RUN apt-get -yqq update
+RUN apt-get -yqq install nodejs npm
+RUN apt-get -yqq install curl
+RUN apt-get -yqq install python3
```

Comment:
See [here](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run) for best practices about apt-get in a Dockerfile. Basically you can do everything in one `RUN` statement that ends with `rm -rf /var/lib/apt/lists/*`.